lib/src/base-styles/_nx-forms.scss

```diff
@@ -10,88 +10,83 @@
 
 .nx-form {
   @include container-vertical;
-  border-top: $nx-border-dark;
-  margin: 0;
-  padding-top: 20px;
+  margin: 0 0 $nx-spacing-xl 0;
```

Comment:
I still think it should have top margin too

Reply:
> I still think it should have top margin too

I have a feeling that this is going to result in us just removing the top margin in a bunch of places (like every tile): https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/f26df0a08f4ce9050772e14ec243743eb6eb9bb7

Reply:
> I have a feeling that this is going to result in us just removing the top margin in a bunch of places (like every tile): f26df0a

That's what `container-vertical` is for